pkg/interface/src/views/components/Dot.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Dot = (props) => {
+  return (<span style={{
+    backgroundColor: props.color || 'currentColor',
+    width: '4px',
+    height: '4px',
+    borderRadius: '50%'
+  }}></span>);
```

Comment:
This should use `Box` and the associated styled system props

Reply:
Seems like you could use `Icon` and `Bullet` or etc.